desktop/main.js

```diff
@@ -201,15 +198,17 @@ const mainWindow = (() => {
 
             // When the user clicks a link that has target="_blank" (which is all external links)
             // open the default browser instead of a new electron window
-            browserWindow.webContents.on('new-window', (e, url) => {
-                // make sure local urls stay in electron perimeter
-                if (url.substr(0, 'file://'.length) === 'file://') {
-                    return;
+            browserWindow.webContents.setWindowOpenHandler(({url}) => {
```

Comment:
The `new-window` event was deprecated and replaced by this `setWindowOpenHandler` API in Electron 12.